lib/erb_lint/linters/deprecated_classes.rb

```diff
@@ -22,18 +22,14 @@ def initialize(config)
         @addendum = config.fetch('addendum', '')
       end
 
-      protected
-
-      def lint_lines(lines)
+      def lint_file(file_tree)
         errors = []
 
-        lines.each_with_index do |line, index|
-          start_tags = StartTagHelper.start_tags(line)
-          start_tags.each do |start_tag|
-            start_tag.attributes.select(&:class?).each do |class_attr|
-              class_attr.value.split(' ').each do |class_name|
-                errors.push(*generate_errors(class_name, index + 1))
-              end
+        html_elements = Parser.filter_erb_nodes(file_tree)
+        html_elements.each do |html_element|
+          html_element.attribute_nodes.select { |attribute| attribute.name.casecmp('class') == 0 }.each do |class_attr|
+            remove_escaped_erb_tags(class_attr.value).split(' ').each do |class_name|
```

Comment:
Same here. Is `remove_escaped_erb_tags(class_attr.value).split(' ')` a collection of `class_names`?

I can figure out what it means after reading closely, but make it easy for me :)